devel/grpc/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup  github 1.0
+PortGroup  cxx11 1.1
```

Comment:
Not required, but it's nice to align everything at 20 spaces :)

Reply:
Yes I surprisingly overlooked these lines. I myself am usually quite picky when it comes to formatting like this.